package.json

```diff
@@ -98,19 +98,19 @@
     "path-to-regexp": "^3.1.0",
     "polyfill-crypto.getrandomvalues": "1.0.0",
     "ramda": "^0.26.1",
-    "react": "^16.10.1",
-    "react-dom": "^16.10.1",
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2",
     "react-helmet": "^5.2.1",
     "react-lazyload": "^2.6.2",
     "react-router-config": "^5.1.1",
-    "react-router-dom": "^5.0.1",
+    "react-router-dom": "5.0.1",
```

Comment:
Is there a reason we aren't using a range for `react-router-dom`?

Reply:
That was my mistake, I was testing something with `react-router-dom` and forgot to remove my change 😅